### PR TITLE
Same-site cookies

### PR DESCRIPTION
**Pull request by markrcote**
Enable same-site cookies, and remove some old deprecated code.

### CODE REVIEW

File: pulseguardian/web.py [hunk 1]
```diff
@@ -63,53 +63,12 @@
...
-    subject = cert.get_subject()
-    subject.CN = cn
-    subject.O = 'Dummy Certificate'
-
-    issuer = cert.get_issuer()
-    issuer.CN = 'Untrusted Authority'
-    issuer.O = 'Self-Signed'
-
-    pkey = crypto.PKey()
-    pkey.generate_key(crypto.TYPE_RSA, 1024)
```
Comment (edmorley) [inline pulseguardian/web.py]: Confirmed -- pulseguardian is using `Werkzeug==0.14.1`, which has now raised to 2048:
https://github.com/pallets/werkzeug/blob/0.14.1/werkzeug/serving.py#L413

File: pulseguardian/web.py [hunk 2]
```diff
@@ -63,53 +63,12 @@
...
-
-# This is used by werkzeug.serving.make_ssl_devcert().
-werkzeug.serving.generate_adhoc_ssl_pair = generate_adhoc_ssl_pair
-
 # Initialize the web app.
 app = Flask(__name__)
 app.config['SERVER_NAME'] = config.flask_server_name
 app.config['PREFERRED_URL_SCHEME'] = ('https' if config.flask_use_ssl
                                       else 'http')
+app.config['SESSION_COOKIE_SAMESITE'] = 'Lax'
```
Comment (edmorley) [inline pulseguardian/web.py]: http://flask.pocoo.org/docs/1.0/config/#SESSION_COOKIE_SAMESITE
http://flask.pocoo.org/docs/1.0/security/#set-cookie-options

Comment (markrcote) [reply pulseguardian/web.py]: What exactly are you saying? :)

Comment (edmorley) [reply pulseguardian/web.py]: Only linking to the docs for reference :-)

**Pull request merged**